packing_product_change/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{'name': 'Product Equivalences',
+ 'version': '10.0.1.0.0',
+ 'license': 'AGPL-3',
+ 'author': "Camptocamp, "
+           "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+           "Odoo Community Association (OCA)",
+ 'category': 'Hidden',
+ 'summary': 'change product in packing',
+ 'website': 'http://www.camptocamp.com',
+ 'depends': ['sale_stock'],
+ 'data': [
+    "wizard/replace_product_view.xml",
+    "views/stock_view.xml",
```

Comment:
packing_product_change/__manifest__.py:17:9: E126 continuation line over-indented for hanging indent

Add 1 space here